src/rosdistro/distribution.py

```diff
@@ -33,11 +33,12 @@
 
 from .manifest_provider.git import git_manifest_provider
 from .manifest_provider.github import github_manifest_provider
+from .manifest_provider.bitbucket import bitbucket_manifest_provider
```

Comment:
Please move the new import above the existing ones to be in alphabetical order (following PEP8).

Same below in the `release.py` file.